src/daemons/StorageDaemon.cpp

```diff
@@ -159,18 +159,32 @@ int main(int argc, char *argv[]) {
 
     auto ioThreadPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_num_io_threads);
 
+    std::string clusteridFile =
+        folly::stringPrintf("%s/%s", paths[0].c_str(), "/storage.cluster.id");
+    auto clusterMan
+        = std::make_unique<nebula::meta::ClusterManager>("", clusteridFile);
+    if (!clusterMan->loadClusterId()) {
+        LOG(INFO) << "storaged misses clusterId";
+    }
+
     // Meta client
     auto metaClient = std::make_unique<nebula::meta::MetaClient>(ioThreadPool,
                                                                  std::move(metaAddrsRet.value()),
                                                                  localhost,
+                                                                 clusterMan.get(),
                                                                  true);
     if (!metaClient->waitForMetadReady()) {
         LOG(ERROR) << "waitForMetadReady error!";
         return EXIT_FAILURE;
     }
     auto gflagsManager = std::make_unique<nebula::meta::ClientBasedGflagsManager>(metaClient.get());
     gflagsManager->init();
-
+    if (!clusterMan->clusterIdDumped()) {
```

Comment:
What's the meaning about the code?

Reply:
I have added a status clusterIdDumped_(return by clusterMan->clusterIdDumped()) representing if clusterId should be dumped(used in mete client); Here, clusterMan->clusterIdDumped() is false, then we should dump clusterId;